### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by kulla**

### CODE REVIEW

File: src/data/de/index.ts [hunk 1]
```diff
@@ -176,6 +177,7 @@ export const instanceData = {
       video: "Video abspielen von %provider%",
       applet: "Applet laden von %provider%",
       twingle: "Spendenformular laden",
+      audio: 'Play audio from %provider%',
```
Comment (CodingDive) [inline src/data/de/index.ts:180]: Still English

File: src/data/de/index.ts [hunk 2]
```diff
@@ -161,7 +161,8 @@ export const instanceData = {
         theorem: "Satz",
         proof: "Beweis"
       },
-      loadingVideoFailed: "Etwas ist schiefgelaufen"
+      loadingVideoFailed: "Etwas ist schiefgelaufen",
+      loadingAudioFailed: 'Something went wrong'
```
Comment (CodingDive) [inline src/data/de/index.ts:165]: Still English

File: src/data/de/index.ts [hunk 3]
```diff
@@ -811,6 +813,11 @@ export const loggedInData = {
           titlePlaceholder: "Titel",
           url: 'URL',
           seoTitle: "Titel für Suchmaschinen"
+        },
+        audio: {
+          title: 'Audio',
+          description: 'Link to audio files on Vocaroo',
+          audioUrl: 'Enter Audio URL'
```
Comment (CodingDive) [inline src/data/de/index.ts:820]: Dachte ich habe es auf Crowdin übersetzt :thinking:

File: src/data/es/index.ts [hunk 4]
```diff
@@ -161,7 +161,8 @@ export const instanceData = {
         theorem: "Teorema",
         proof: "Prueba"
       },
-      loadingVideoFailed: "Algo ha fallado"
+      loadingVideoFailed: "Algo ha fallado",
+      loadingAudioFailed: 'Something went wrong'
```
Comment (CodingDive) [inline src/data/es/index.ts:165]: Still English

File: src/data/es/index.ts [hunk 5]
```diff
@@ -176,6 +177,7 @@ export const instanceData = {
       video: "Reproducir vídeo de %provider%",
       applet: "Cargar Applet de %provider%",
       twingle: "Cargar formulario de donación",
+      audio: 'Play audio from %provider%',
```
Comment (CodingDive) [inline src/data/es/index.ts:180]: Still English

File: src/data/es/index.ts [hunk 6]
```diff
@@ -811,6 +813,11 @@ export const loggedInData = {
           titlePlaceholder: "Título",
           url: 'URL',
           seoTitle: "Título para motores de búsqueda"
+        },
+        audio: {
+          title: 'Audio',
+          description: 'Link to audio files on Vocaroo',
+          audioUrl: 'Enter Audio URL'
```
Comment (CodingDive) [inline src/data/es/index.ts:820]: Still English

File: src/data/fr/index.ts [hunk 7]
```diff
@@ -161,7 +161,8 @@ export const instanceData = {
         theorem: 'Theorem',
         proof: 'Proof'
       },
-      loadingVideoFailed: 'Something went wrong'
+      loadingVideoFailed: 'Something went wrong',
+      loadingAudioFailed: 'Something went wrong'
```
Comment (CodingDive) [inline src/data/fr/index.ts:165]: Still English

File: src/data/fr/index.ts [hunk 8]
```diff
@@ -176,6 +177,7 @@ export const instanceData = {
       video: "Jouer la vidéo de %provider%",
       applet: "Charger l'Applet de %provider%",
       twingle: "Charger le formulaire de don",
+      audio: 'Play audio from %provider%',
```
Comment (CodingDive) [inline src/data/fr/index.ts:180]: Still English

File: src/data/fr/index.ts [hunk 9]
```diff
@@ -811,6 +813,11 @@ export const loggedInData = {
           titlePlaceholder: "Titre",
           url: 'URL',
           seoTitle: "Titre pour les moteurs de recherche"
+        },
+        audio: {
+          title: 'Audio',
+          description: 'Link to audio files on Vocaroo',
+          audioUrl: 'Enter Audio URL'
```
Comment (CodingDive) [inline src/data/fr/index.ts:820]: Still English

File: src/data/hi/index.ts [hunk 10]
```diff
@@ -161,7 +161,8 @@ export const instanceData = {
         theorem: 'Theorem',
         proof: 'Proof'
       },
-      loadingVideoFailed: 'Something went wrong'
+      loadingVideoFailed: 'Something went wrong',
+      loadingAudioFailed: 'Something went wrong'
```
Comment (CodingDive) [inline src/data/hi/index.ts:165]: Still English

File: src/data/hi/index.ts [hunk 11]
```diff
@@ -176,6 +177,7 @@ export const instanceData = {
       video: "से वीडियो चलाएं %provider%",
       applet: "%provider% से लोड एप्लेट ",
       twingle: "लोड दान प्रपत्र",
+      audio: 'Play audio from %provider%',
```
Comment (CodingDive) [inline src/data/hi/index.ts:180]: Still English

File: src/data/hi/index.ts [hunk 12]
```diff
@@ -811,6 +813,11 @@ export const loggedInData = {
           titlePlaceholder: "शीर्षक",
           url: 'URL',
           seoTitle: 'Title for search engines'
+        },
+        audio: {
+          title: 'Audio',
+          description: 'Link to audio files on Vocaroo',
+          audioUrl: 'Enter Audio URL'
```
Comment (CodingDive) [inline src/data/hi/index.ts:820]: Still English

File: src/data/ta/index.ts [hunk 13]
```diff
@@ -161,7 +161,8 @@ export const instanceData = {
         theorem: 'Theorem',
         proof: 'Proof'
       },
-      loadingVideoFailed: 'Something went wrong'
+      loadingVideoFailed: 'Something went wrong',
+      loadingAudioFailed: 'Something went wrong'
```
Comment (CodingDive) [inline src/data/ta/index.ts:165]: Still English

File: src/data/ta/index.ts [hunk 14]
```diff
@@ -176,6 +177,7 @@ export const instanceData = {
       video: 'Play Video from %provider%',
       applet: 'Load Applet from %provider%',
       twingle: 'Load Donation Form',
+      audio: 'Play audio from %provider%',
```
Comment (CodingDive) [inline src/data/ta/index.ts:180]: Still English

File: src/data/ta/index.ts [hunk 15]
```diff
@@ -811,6 +813,11 @@ export const loggedInData = {
           titlePlaceholder: "தலைப்பு",
           url: 'URL',
           seoTitle: 'Title for search engines'
+        },
+        audio: {
+          title: 'Audio',
+          description: 'Link to audio files on Vocaroo',
+          audioUrl: 'Enter Audio URL'
```
Comment (CodingDive) [inline src/data/ta/index.ts:820]: Still English

**Review by CodingDive — APPROVED** [review]
Translation and neither my input in crowdin worked that well, but I want to fix staging which without this throws `TypeError: Cannot read properties of undefined (reading 'audioUrl')` for non English languages

**Comment by elbotho** [comment]
> Translation and neither my input in crowdin worked that well, but I want to fix staging which without this throws `TypeError: Cannot read properties of undefined (reading 'audioUrl')` for non English languages

What? :o
All strings should fall back to `en` so there should never be errors when strings are not yet translated (and I never saw an error like this).

For crowdin: the strings are not translated for me. Only `audioUrl` seems to be. Not sure what happened / what you did. Did you click save / press enter after each entry?
image
The GitHub Integration periodically fetches new translations and opens a new PR. You can trigger a refresh here:
https://crowdin.com/project/serlo/apps/system/github (but I'm not sure if you need special rights)

**Pull request merged**